src/common/mempool.cc

```diff
@@ -95,9 +97,21 @@ size_t mempool::pool_t::allocated_items() const
 
 void mempool::pool_t::adjust_count(ssize_t items, ssize_t bytes)
 {
-  thread_shard_index = (thread_shard_index == num_shards) ? pick_a_shard_int() : thread_shard_index;
-  shard[thread_shard_index].items += items;
-  shard[thread_shard_index].bytes += bytes;
+#if !defined(_GNU_SOURCE) && defined(WITH_SEASTAR) && !defined(WITH_ALIEN)
```

Comment:
Oops, it looks the logic is flapped – crimson (`WITH_SEASTAR`) uses the old, `thread_local`-based bits; the world ex crimson uses the new things (cpu local).

Reply:
Should be fixed.